src/components/Scoreboard/Scoreboard.js

```diff
@@ -23,7 +23,7 @@ const Scoreboard = () => {
   const [solutions, setSolutions] = useState([]);
 
   const updateSolutions = async () => {
-    const response = await get('solutions/');
+    const response = await get('scoreboard/');
```

Comment:
This file is out of scope of this task. You may change it locally, but do not commit this changes to this branch. In order to fix this, use `git commit --amend` and then force-push (`-f`) your branch.